pkg/web_app/lib/src/_dom_helper.dart

```diff
@@ -209,14 +221,24 @@ Element _buildDialog(
                       ..children = [
                         Element.span()
                           ..classes.add('mdc-button__label')
-                          ..innerText = 'Ok',
+                          ..innerText = okButtonText ?? 'Ok',
                       ],
                   ],
               ],
           ],
         Element.div()..classes.add('mdc-dialog__scrim'),
       ];
 
+/// Creates an [Element] with unformatted [text] content.
+Element text(String text) => Element.div()..text = text;
+
+/// Creates an [Element] with Markdown-formatted content.
+Element markdown(String text) => Element.div()
+  ..setInnerHtml(
+    md.markdownToHtml(text),
```

Comment:
This uses `sanitize_html` right?

Reply:
This is just `dart:html`'s `innerHtml` setter's validator. However the markdown source is either known or coming from our backend. We could add extra checks but probably non-blocking to this PR.